CHANGELOG.md

```diff
@@ -15,6 +15,14 @@ You can find its changes [documented below](#051---2024-07-04).
 
 This release supports Bevy version 0.14 and has an [MSRV][] of 1.80.
 
+## [0.6.1] - 2024-08-14
+
+This release supports Bevy version 0.14 and has an [MSRV][] of 1.80.
+
+### Fixed
+
+- Text is now properly skipped when a `SkipEncoding` component is present.
+
```

Comment:
Again with the merging of a release into an unrelated PR...

Reply:
The entire release _is_ the PR. I'm not sure how it would be unrelated, so it feels pedantic to split this up, but sure.

Reply:
Well I did quite clearly say in my review comment that I'd be happy to merge this PR as-it-was. But only if it was packaged as a PR which is doing a release, which was not being clear in doing so.